src/coreclr/tools/Common/TypeSystem/Ecma/EcmaMethod.cs

```diff
@@ -353,6 +353,14 @@ public override bool IsDefaultConstructor
             }
         }
 
+        public override bool IsPublic
+        {
+            get
+            {
+                return Attributes.IsPublic();
```

Comment:
Does this need to take assembly visibility into account as well?

Reply:
If we make it so, I think it would be wrong the for the interface resolution case - ECMA-335 II.12.2 is only concerned about the accessibility of the method and all that matters is that method is public.

Reply:
ok